pages/404.html

```diff
@@ -5,9 +5,40 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="../css/main.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src='../assets/js/main.js'></script>
     <title>Document</title>
 </head>
 <body>
+    <div data-include= "header-b"></div>
```

Comment:
@Joggz I think you meant to use `header` here and not `header-b` because this version of this header has the back button. Also, you should change the background color for this header.

```
.404-page {
    .header {
     background-color: none;
}
}
```

Reply:
@SEUNAGBEYE thanks for the review, I have changed the reusable header to the header but @Techgurll  styled the background of the reusable  header except I am supposed to go alter her codes